<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>结构配置</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>971</width>
     <height>721</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>基础</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>881</width>
       <height>300</height>
      </rect>
     </property>
     <property name="title">
      <string>大地导电率</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>442</width>
        <height>79</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1" rowspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>1/(Ω·cm)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>1/(Ω·cm)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" rowspan="2">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton_4">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>土壤类型</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_rou">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="toolTipDuration">
            <number>0</number>
           </property>
           <property name="statusTip">
            <string>选择地质情况</string>
           </property>
           <property name="whatsThis">
            <string>选择地质情况</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText">
            <string/>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButton_5">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>自定义</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_rou">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>自定义导电率</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>区段</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_init">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>m  -</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_end">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>40</y>
        <width>16</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>m</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_rou">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>331</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addrou">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>250</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delrou">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>250</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>881</width>
       <height>200</height>
      </rect>
     </property>
     <property name="title">
      <string>供电臂</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>供电臂长度</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_sectionlength">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>200</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>请填写供电臂长度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>21</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>km</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>基本分段长度</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_deltalength">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>200</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>请填写基本分段长度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>21</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>km</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_save1">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>640</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>导线选择</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>441</width>
       <height>601</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_save2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>640</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_check">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>31</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>查
看
参
数</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>横向元件</string>
    </attribute>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>401</width>
       <height>561</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>分类</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>图像</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>电源</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <item>
       <property name="text">
        <string>牵引变压器</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>C:/Users/ZhuMing/.designer/backup/test.svg</normaloff>C:/Users/ZhuMing/.designer/backup/test.svg</iconset>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>横向并联线</string>
      </property>
      <item>
       <property name="text">
        <string>上下行并联线</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>保护线pw1-钢轨ra1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>综合地线e1-钢轨ra1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>保护线pw2-钢轨ra3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>综合地线e2-钢轨ra3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>钢轨ra1-大地g</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>钢轨ra3-大地g</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>综合地线e1-大地g</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>综合地线e2-大地g</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>自耦变压器</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>并联电阻</string>
      </property>
     </item>
    </widget>
    <widget class="QTableView" name="tableView_element">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>50</y>
       <width>461</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_del">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>560</y>
       <width>61</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>560</y>
       <width>71</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>+/确定</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
